spec/rspec/core_spec.rb

```diff
@@ -256,7 +270,7 @@ def reporter
 
       RSpec.clear_examples
 
-      RSpec.configuration.deprecation_stream = StringIO.new(deprecations = "".dup)
+      RSpec.configuration.deprecation_stream = StringIO.new(deprecations = +"")
```

Comment:
Can you explain this?

Reply:
This would break pre-2.0 builds. We still maintain 1.8.7+ compatibility in RSpec 3.x.

Reply:
Yeah my rubocop (which is wrong) told me to do it. Will revert.

Reply:
```
rubocop: warning
Performance/UnfreezeString - Use unary plus to get an unfrozen string literal.
```

But there is no magic comment nor are we in a ruby version w/ frozen strings by default so why? Idk.